src/models/topological_elements.jl

```diff
@@ -19,7 +19,17 @@ Bus(;   number = 0,
         basevoltage = nothing
     ) = Bus(number, name, bustype, angle, voltage, PowerSystems.orderedlimits(voltagelimits, "Voltage"), basevoltage)
 
-struct LoadZones
+struct LoadZones  <: PowerSystemDevice
     number::Int
     name::String
+    buses::Array{PowerSystems.Bus,1}
+    maxrealpower::Float64
```

Comment:
Is this a maxrealpower transfer? @claytonpbarrows what is the common use of load zones?

Reply:
no, I think it's peak load